test/1_api.js

```diff
@@ -130,20 +132,21 @@ describe('Server', function() {
 	});
 
 	afterEach(function() {
-		xhr.restore();
-		clock.restore();
-		server.createScript.restore();
+		if (typeof xhr.restore === 'function') {
```

Comment:
https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=Attempted+to+wrap+which+is+already+wrapped

To be honest I only know that this causes flakiness but I don't know why. I like this check for safety reasons.